### ISSUE TITLE
investigate !!!

### ISSUE THREAD

**Issue by longto** [longto/l4lsite#7]
CSS : table with fixed header
http://www.imaputz.com/cssStuff/bigFourVersion.html#

**Comment by longto** [comment]
http://www.w3schools.com/angular/angular_directives.asp
The ng-model Directive
The ng-model directive binds the value of HTML controls (input, select, textarea) to application data.

The ng-model directive can also:

Provide type validation for application data (number, email, required).?
Provide status for application data (invalid, dirty, touched, error).?
Provide CSS classes for HTML elements.
Bind HTML elements to HTML forms.

**Comment by longto** [comment]
http://www.w3schools.com/angular/angular_tables.asp
angularJs table , loop thought table ?
<!DOCTYPE html>
<html>
<style>
table, th , td  {
  border: 1px solid grey;
  border-collapse: collapse;
  padding: 5px;
}
table tr:nth-child(odd)	{
  background-color: #f1f1f1;
}
table tr:nth-child(even) {
  background-color: #ffffff;
}
</style>
<script src= "http://ajax.googleapis.com/ajax/libs/angularjs/1.3.14/angular.min.js"></script>
<body>

<div ng-app="myApp" ng-controller="customersCtrl"> 

<table>
  <tr ng-repeat="x in names">
      <td ng-repeat="y in cols">{{x.[y]}}</td>
  </tr>
</table>

</div>

<script>
var app = angular.module('myApp', []);
app.controller('customersCtrl', function($scope, $http) {
    $http.get("http://www.w3schools.com/angular/customers.php")
    .success(function (response) {$scope.names = response.records;
$scope.cols=["Name","Country"];
});
});
</script>

</body>
</html>

**Comment by longto** [comment]
Encode Json to UTF8 with vietnamese

**Comment by longto** [comment]
angular-seed 
http://code.tutsplus.com/tutorials/creating-a-web-app-from-scratch-using-angularjs-and-firebase--cms-22391

**Comment by longto** [comment]
https://github.com/angular/angular.js/wiki/Understanding-Scopes
investigate scope

**Comment by longto** [comment]
http://stackoverflow.com/questions/17470790/how-to-use-a-keypress-event-in-angularjs
define new directive enter key press

**Comment by longto** [comment]
define service in create module app

**Comment by longto** [comment]
gui CV cho trong. long

**Comment by longto** [comment]
referrence https://addition-it.com/contact

**Comment by longto** [comment]
CK editor
http://doksoft.com/soft/ckeditor-bootstrap/demo.html

**Comment by longto** [comment]
display html content on angularJs, keep from text editor

**Comment by longto** [comment]
upload file to amazon s3
http://www.cheynewallace.com/uploading-to-s3-with-angularjs/

**Comment by longto** [comment]
skip upload file later :( build layout for web

**Comment by longto** [comment]
compare cloudinary and amazon S3 in upload images/media/files to choose which one is better

**Comment by longto** [comment]
using grunt as strust http://stackoverflow.com/questions/24097146/include-all-javascript-files-in-a-directory-to-a-html-file-in-angularjs-with-gr

**Comment by longto** [comment]
layout 
http://demo.yoarts.com/start/
